src/docs/frontend/storybook.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Storybook Styleguide
```

Comment:
What's leftover in the Notion doc:

```
## **Relevant Resources**

- Feature request: [https://github.com/getsentry/ops-req/issues/35](https://github.com/getsentry/ops-req/issues/35)
- PR with implementation: [https://github.com/getsentry/sentry/pull/8412](https://github.com/getsentry/sentry/pull/8412)
```

@billyvg is this important information?

Reply:
Hmm not really, I mean there's some good information in the implementation PR